ui/src/components/search/SearchWrapper.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Component from 'react-component-component';
-// import { Arranger } from '@arranger/components/dist/Arranger';
 // import '@arranger/components/public/themeStyles/beagle/beagle.css';
```

Comment:
Added this CSS in next PR (affects Models page): https://github.com/nci-hcmi-catalog/portal/pull/1101

Reply:
Added CSS in this PR to insure Pagination is rendering correctly